### DISCUSSION TITLE
Logged Fingerprint is different in Browser without Incognito but same in normal browser & your demo site

### DISCUSSION THREAD

**Discussion by deadcoder0904** [thumbmarkjs/thumbmarkjs#14] [General]
So my fingerprint on [your demo site](https://www.thumbmarkjs.com/) & my Incognito Chrome browser is same.

But it is different in my Chrome browser without Incognito which is weird.

I was expecting it to be same on your demo site & my own browser.

Anyway I can help you debug?

Another libraries like https://github.com/Rajesh-Royal/Broprint.js & https://github.com/abrahamjuliot/creepjs gave same output for my normal browser & incognito.

**Comment by ilkkapeltola** [comment]
What you can do to debug is, open the demo page with the two different fingerprints.

Then open this page:
https://jsonformatter.org/json-compare

Use the copy to clipboard button on the demo page to copy the two different JSONs over to the compare page. Then let me know what the differences are.

You could also tell me the two different IDs and I'll see if I can find them in my logs.

Thank you!

**Reply by deadcoder0904** [reply]
now, they are same.

i didn't find them same last week on my own site.

i didn't test it on https://www.thumbmarkjs.com/ though but if i face it again, i'll let you know.

is there a way to log the whole json tree like you do on your website?

**Reply by ilkkapeltola** [reply]
Yeah you can call 
`getFingerprintData()`
